.github/ISSUE_TEMPLATE/dataset_review.yaml

```diff
@@ -0,0 +1,53 @@
+name: Dataset Review Request
+description: Request Reviews & Comments on Your Dataset
+title: "[datset]: "
+labels: ["data help"]
+assignees:
+  - abenson
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this form to request comments from the community on your DwC-formmatting attempt.
```

Comment:
I know this will be open to all for review, but I'm wondering if saying requesting comments from the community will scare folks off?

Maybe "Use this form to receive feedback on your work aligning data to Darwin Core." ? Something a little more vague is what I'm going after.